### PR TITLE
Feature/new urls

### PR DESCRIPTION
**Pull request by prembonsall**
## Description
Added URLs for testing sessions

### CODE REVIEW

File: app/views/index.html [hunk 1]
```diff
@@ -26,9 +26,25 @@ <h1>Urgent Care Signposting Prototypes</h1>
...
+                    </li>
+                    <li>
+                    <p>Interruptor & Pete</p>
+                        <ol>
+                            <li><a href="/google/search-ae">Google journey A&E</a></li>
+                            <li><a href="/ae-finder">Direct to A&E finder</a></li>
+                        </ol>
+                    </li> 
                     <li><a href="/carecarder">Care Carder</a> </li>
+                    <li><a href="https://www.nhs.uk/conditions/diabetic-eye-screening-approach-1/">Diabetic eye screening test approach 1</a></li>
```
Comment (jbroodie) [inline app/views/index.html]: What are these links for?

File: app/views/google/results-wi.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<html>
+    <head>
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    </head>
+    <body>
+      <a href="/ae-finder"><img src="/assets/images/google-search/google-results-ae.png" style="width: 100%; display: block;"></a>
```
Comment (jbroodie) [inline app/views/google/results-wi.html]: Do we need a walk-in centre image for this rather?

Comment (jbroodie) [reply app/views/google/results-wi.html]: And should this go to the walk-in-finder?

Comment (prembonsall) [reply app/views/google/results-wi.html]: Yeah, still working on the google pages atm, these changes were to just set up different links on the index page. One for walk in one for A&E

File: app/views/index.html (hunk 1 above)
Comment (prembonsall) [reply app/views/index.html]: Mark and Ana asked me to add them to the index so they could have all the things they're testing in one place

**Pull request closed without merging**